CONTRIBUTING.md

```diff
@@ -1,5 +1,24 @@
+## Contributing Code
+We highly value contributions made from the community and we thank you for stopping by to learn more about how to contribute to this connector.
 
-# Contributing
+### Hard Requirements
+- Please include a test for any code you submit via PR. 
+
+### The Basics
+- Create your branch from dev
+- Open your PR against the dev branch
+- Once you open your PR, it will automatically be assigned to the correct review group 
+(https://github.com/orgs/microsoft/teams/kafka-connect-cosmosdb-devs)
```

Comment:
Should the link be shown fully here, or would it be better to wrap the link inside `correct review group` (similar to `create a bug report` below)?